bitnami/grafana-operator/templates/deployment.yaml

```diff
@@ -0,0 +1,106 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "grafana-operator.fullname" . }}
+  labels: {{- include "grafana-operator.labels" . | nindent 4 }}
+spec:
+  replicas: {{ .Values.replicaCount }}
+  selector: 
+    matchLabels: {{- include "grafana-operator.matchLabels" . | nindent 6 }}
+  {{- if .Values.updateStrategy }}
+  strategy: {{ toYaml .Values.updateStrategy | nindent 4 }}
+  {{- end }}
+  template:
+    metadata:
+      labels: {{- include "grafana-operator.labels" . | nindent 8 }}
+      {{- if .Values.podAnnotations }}
+      annotations: {{- toYaml .Values.podAnnotations | nindent 8 }}
+      {{- end }}
+    spec: 
+      {{- include "grafana-operator.imagePullSecrets" . | nindent 6 }}
+      {{- if .Values.nodeSelector }}
+      nodeSelector: {{- include "grafana-operator.tplValue" (dict "value" .Values.nodeSelector "context" $) | nindent 8 }}
+      {{- end }}
+      {{- if .Values.affinity }}
+      affinity: {{- include "grafana-operator.tplValue" (dict "value" .Values.affinity "context" $) | nindent 8 }}
+      {{- end }}
+      {{- if .Values.tolerations}}
+      tolerations: {{- include "grafana-operator.tplValue" (dict "value" .Values.tolerations "context" $) | nindent 8 }}
+      {{- end }}
+      serviceAccountName: {{ include "grafana-operator.serviceAccountName" . }}
+      {{- if .Values.securityContext.enabled }}
+      securityContext:
+        runAsUser: {{ .Values.securityContext.runAsUser }}
+        fsGroup: {{ .Values.securityContext.fsGroup }}
+        runAsNonRoot: {{ .Values.securityContext.runAsNonRoot }}
+        runAsGroup: {{ .Values.securityContext.runAsGroup }}
+        {{- if .Values.securityContext.supplementalGroups }}
+        supplementalGroups:
+        {{- with .Values.securityContext.supplementalGroups }}
+          - {{ . }}
+        {{- end }}
+        {{- end }}
+      {{- end }}
+      containers:
+        - name: grafana-operator
+          image: {{ include "grafana-operator.image" . }}
+          imagePullPolicy: {{ .Values.image.pullPolicy }}
+          ports:
+          - containerPort: 8080
+            name: metrics
+          command:
+          - grafana-operator
+          args:
+          - -grafana-image={{ include "grafana-operator.grafana.baseImage" . }}
+          - -grafana-image-tag={{ .Values.grafana.image.tag | quote}}
+          - -grafana-plugins-init-container-image={{ include "grafana-operator.pluginInit.baseImage" . }}
+          - -grafana-plugins-init-container-tag={{ .Values.grafanaPluginInit.image.tag | quote}}
```

Comment:
Failed to apply default image tag "quay.io/integreatly/grafana_plugins_init:\"0.0.3\"": couldn't parse image reference "quay.io/integreatly/grafana_plugins_init:\"0.0.3\"": invalid reference format

tested with k8s v.1.17.5

Reply:
@1337andre I fixed the templating bug.
Could you please try again?